### PR TITLE
 Change `escapedMetricsNameCache` method to support `/`

### PR DESCRIPTION
**Pull request by yswdqz**
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [ ] Update the documentation to include this new feature.
- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by yswdqz** [comment]
Initially I wanted to convert `/` to `-` but mal doesn't support `-` either.
I also tried to write unit tests, but creating PrometheusMetricConverter object was complicated. So I have tested it locally.

File: oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/PrometheusMetricConverter.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public String load(final String name) {
 
     public PrometheusMetricConverter(Rule rule, MeterSystem service) {
         this.convert = new MetricConvert(rule, service);
-        this.metricsNameEscapePattern = Pattern.compile("\\.");
+        this.metricsNameEscapePattern = Pattern.compile("[/.]");
```
Comment (wu-sheng) [inline oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/PrometheusMetricConverter.java:74]: I am not good at regex, is this testes? Do we have UT for this?

**Review by kezhenxu94 — APPROVED** [review]
LGTM

File: oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/PrometheusMetricConverter.java (hunk 1 above)
Comment (yswdqz) [reply oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/prometheus/PrometheusMetricConverter.java:74]: I have tried to write unit tests, but creating PrometheusMetricConverter object was complicated. So I have tested it locally.

**Pull request merged**